pkg/ibmvpc/vpc.go

```diff
@@ -395,12 +399,25 @@ func (sgl *SecurityGroupLayer) RulesInConnectivity(src, dst vpcmodel.Node,
 	return res, nil
 }
 
+// IsDefault the layer is considered default if all sgs' rules are default
+func (sgl *SecurityGroupLayer) IsDefault() bool {
```

Comment:
when referring to a method in documentation, please add () (should be IsDefault() )
fix also in other other methods documentation

Reply:
my GoLand editor insists I add the function name without the ()
I agree with you, but never messes with GoLand